### PR TITLE
[services] improve startup time by changing startup order

### PR DESCRIPTION
**Pull request by stepanblyschak**
Signed-off-by: Stepan Blyschak <stepanb@mellanox.com>

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: files/build_templates/dhcp_relay.service.j2 [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=DHCP relay container
-Requires=updategraph.service swss.service teamd.service
-After=updategraph.service swss.service teamd.service
+Requires=updategraph.service swss.service syncd.service teamd.service
```
Comment (yxieca) [inline files/build_templates/dhcp_relay.service.j2]: Why is Requires needed? Is After not enough?

Comment (jleveque) [reply files/build_templates/dhcp_relay.service.j2]: `Requires` will start the service ONLY if the dependency is active and has successfully started. `After` is more of an order-only dependency and will start the service after the dependency should start, regardless of if it is enabled or successfully started.

Comment (yxieca) [reply files/build_templates/dhcp_relay.service.j2]: I understand how Requires work. That wasn't my question.

My question was that why did syncd added to required for dhcp_relay but not any other services below. Can you explain?

Comment (stepanblyschak) [reply files/build_templates/dhcp_relay.service.j2]: This wasn't my intention. This change was made by mistake. I reverted it.
This PR is only about order

**Pull request merged**